app/repositories/PublishedRepository.scala

```diff
@@ -140,4 +141,17 @@ class PublishedRepositoryImpl @Inject() (mongoComponent: ReactiveMongoComponent)
     //$COVERAGE-ON$
   }
 
+  def delete(id: String): Future[RequestOutcome[String]] = {
+    val selector = Json.obj("_id" -> id)
+    collection.findAndRemove(selector)
```

Comment:
generates "method findAndRemove in trait GenericCollection is deprecated (since 0.18.0): Use the other `findAndRemove`"